week-3/db/db.go

```diff
@@ -0,0 +1,56 @@
+package db
+
+import (
+	"fmt"
+	"log"
+	"sync"
+
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/mysql" //mysql dialect
+)
+
+var (
+	once     sync.Once
+	instance *MySQLClient
+)
+
+const (
+	username = "feedback_user"
+	password = "password"
+	dbname   = "feedback"
+)
+
+// MySQLClient ...
+type MySQLClient struct {
+	*gorm.DB
+}
+
+// PassengerFeedback ...
+type PassengerFeedback struct {
+	BookingCode string `gorm:"PRIMARY_KEY"`
+	PassengerID int32  `gorm:"NOT NULL"`
```

Comment:
`PassengerID` must be indexed, 
moreover, we should have standard columns like: `id`, `createdAt` and `updatedAt`